### PR TITLE
#2725 Blank spaces and lines added or removed, points instead of comma…

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes [#2725](https://github.com/dynawo/dynawo/issues/2725)

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/DriveTrainPmConstant.mo [hunk 1]
```diff
@@ -29,6 +29,5 @@ equation
     <p> This model is a simplified model for the purpose of emulating the behavior of torsional mode oscillations. The shaft damping coefficient (Dshaft) in the drive-train model is fitted to capture the net damping of the torsional mode seen in the post fault electrical power response. In the actual equipment, the drive train oscillations are damped through filtered signals and active damping controllers, which obviously are significantly different from the simple generic two mass drive train model used here. Therefore, the
     parameters (and variables) of this simple drive-train model cannot necessarily be compared with
     actual physical quantities directly. </p>
-    <p>In this version A of the drive train, the mechanical power is set to be constant as per the initial electrical power. </p></html>"),
-    Icon);
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/DriveTrainPmConstant.mo:33]: Why this removal ?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/BaseConverters/RefFrameRotation.mo [hunk 2]
```diff
@@ -53,6 +53,5 @@ equation
 
   annotation(
     preferredView = "text",
-    Diagram,
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/BaseConverters/RefFrameRotation.mo:56]: same question

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/WT4Ainjector.mo [hunk 3]
```diff
@@ -152,6 +152,5 @@ equation
 
   annotation(
     preferredView = "diagram",
-    Icon(graphics = {Rectangle(fillColor = {255, 255, 255}, fillPattern = FillPattern.Solid, extent = {{-100, 100}, {100, -100}}), Text(origin = {0, 30}, extent = {{-90, -30}, {90, 30}}, textString = "IEC WT4A"), Text(origin = {0, -30}, extent = {{-90, -30}, {90, 30}}, textString = "Converter")}, coordinateSystem(initialScale = 0.1)),
-  Diagram);
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/WT4Ainjector.mo:156]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo [hunk 4]
```diff
@@ -64,6 +64,5 @@ equation
   end if;
 
   annotation(preferredView = "text",
-    Diagram,
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo:67]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/DriveTrainPmConstant.mo (hunk 1 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/DriveTrainPmConstant.mo:33]: Line 32 ought to remain (I have restored it), line 33 is useless.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/BaseConverters/RefFrameRotation.mo (hunk 2 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/BaseConverters/RefFrameRotation.mo:56]: Useless line.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/WT4Ainjector.mo (hunk 3 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/WT4Ainjector.mo:156]: Line 155 has been restored, line 156 is useless.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo (hunk 4 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo:67]: Useless line.

**Review by cossartque — APPROVED** [review]

**Pull request merged**